drivers/interrupt_controller/Kconfig.clic

```diff
@@ -17,6 +18,13 @@ config NRFX_CLIC
 	help
 	  Interrupt controller for Nordic VPR cores.
 
+config CLIC_SMCLICSHV_EXT
+	bool
+	help
+	  The selective hardware vectoring extension gives users the flexibility
+	  to select the behavior for each interrupt. The CLIC driver needs to
+	  implementat the riscv_clic_irq_vector_set() function.
```

Comment:
```suggestion
	  implement the riscv_clic_irq_vector_set() function.
```

Reply:
Thank you for catching this typo. Fixed it.